code/client/src/Client.cpp

```diff
@@ -0,0 +1,87 @@
+#include <Client.h>
+
+bool Client::updateModule(){
+//    sendRequest();
```

Comment:
The client should foresee a mode for sending requests continuously.
You might want to add a flag for that in Thrift.